open-api/src/main/resources/dipa-hub.yaml

```diff
@@ -48,7 +50,7 @@ paths:
     $ref: "paths/ical.yml"
   /timelines/{timelineId}/milestones:
     $ref: "paths/milestones.yml"
-  /timelines/{timelineId}/milestones/{milestoneId}:
+  /timelines/{timelineId}/milestones/{milestoneId}/deleteMilestone:

```

Comment:
hier gilt das gleiche wie für deleteProject: kann das nicht als delete-Fall in milestones.yml?